posthog/api/survey.py

```diff
@@ -444,6 +448,7 @@ def surveys(request: Request):
     surveys = SurveyAPISerializer(
         Survey.objects.filter(team_id=team.id)
         .exclude(archived=True)
+        .exclude(end_date__gt=date.today())
```

Comment:
this doesn't seem right to me, why are we excluding these surveys? Or am I reading this wrong and it means end date is before today

Reply:
fixed now, no idea why this was there.